homeassistant/components/apprise/notify.py

```diff
@@ -0,0 +1,69 @@
+"""Apprise platform for notify component."""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+
+from homeassistant.components.notify import (
+    ATTR_TITLE,
+    ATTR_TITLE_DEFAULT,
+    PLATFORM_SCHEMA,
+    BaseNotificationService,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_FILE = "config"
+CONF_URL = "url"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Optional(CONF_URL): vol.All(cv.ensure_list, [str]),
+        vol.Optional(CONF_FILE): cv.string,
+    }
+)
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the Apprise notification service."""
+    from apprise import Apprise, AppriseConfig
```

Comment:
Please move this import to the top of the file

Reply:
Done.  It was originally placed in `get_service()` to follow the same structure as other notification services (such as pushbullet, discord, etc).  By keeping it on top of the file, don't you risk throwing an import error every time this file is referenced (even for those who don't plan on using it)?

Anyway; i still made the change :slightly_smiling_face:  as requested.

Reply:
It used to be the other way around, however, not all has been migrated yet.